frontend/federation-cup-athletics-explorer/federation.css

```
@@ -0,0 +1,499 @@
+.fed-main {
+    background-color: var(--bg-primary, #0c1220);
+    color: var(--text-primary, #e8eaed);
+    font-family: 'Outfit', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)